src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -219,9 +219,9 @@ public boolean awaitTermination(long timeout, TimeUnit units) throws Interrupted
     }
 
     @Override
-    public Command loadCommand(int commandStoreId, TxnId txnId, RedundantBefore redundantBefore, DurableBefore durableBefore)
+    public Command loadCommand(int store, TxnId txnId, RedundantBefore redundantBefore, DurableBefore durableBefore)
```

Comment:
it would be nice to stick to consistent naming everywhere, and we mostly refer to the referred objects as `commandStore`, and so `commandStoreId` is most consistent